iotit.go

```diff
@@ -175,6 +182,16 @@ func initCommands() {
 		}
 	}
 
+	clean := func() {
+		w := workstation.NewWorkStation()
+		if err := w.CleanDisk(); err != nil {
+			fmt.Println("[-] Error:", err)
+			return
+		} else {
```

Comment:
__[golint]__
* [golint] if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

<!-- Comment by [SideCI](https://sideci.com) -->